xarray_user_forum_2021/Xarray_User_Summit.ipynb

```diff
@@ -0,0 +1,9690 @@
+{
```

Comment:
<p>not sure if it should be "we're commited" or "we've continued" but current wording feels strange.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_misc/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>We've continued to integrate dask functionality to speed up diagnostic processing or for <code>az.InferenceData</code> objects that won't fit in memory.</p><p> </p><p>Is what I meant, but yea typo on my end </p>